repos: update for kirkstone branch

dylan-matthew-turner opened the pull request:
We will want to perform upstream merges for kirkstone using the upstream merge script. However, for this to work, repos.conf needs to be set up to point to the kirkstone branches of upstreams

chaitu236 commented on scripts/dev/upstream_merge/repos.conf:
```diff
@@ -1,13 +1,14 @@
+sources/meta-qt5-extra       https://github.com/schnitzeltony/meta-qt5-extra       kirkstone             nilrt/master/kirkstone
+sources/meta-rauc            https://github.com/rauc/meta-rauc                     kirkstone             nilrt/master/kirkstone
+sources/pyrex                https://github.com/garmin/pyrex                       master                nilrt/master/kirkstone
+sources/meta-mingw           https://git.yoctoproject.org/meta-mingw               kirkstone             nilrt/master/kirkstone
```
We seem to have [removed](https://github.com/ni/nilrt/commit/68a63c866b9de1b4fd23ce36306e7114f29c6945) `meta-mingw` layer from hardknott and [added it back](https://github.com/ni/nilrt/commit/031b41a6a392b03f54e141c902144b21ceed82c0) but we don't have it listed in hardknott's `repos.conf`. Do you mind adding it?

dylan-matthew-turner replied:
Can do

dylan-matthew-turner replied:
https://github.com/ni/nilrt/pull/208

dylan-matthew-turner replied:
The local branch for meta-mingw appears to be wrong according to the upstream merge script. I'll correct it

dylan-matthew-turner replied:
https://github.com/ni/nilrt/pull/209

amstewart reviewed (changes requested):
[`s/kirstone/kirkstone/`](https://github.com/ni/nilrt/pull/207/commits/d3a1e03f38f2df501aec23155fadfd4e0f427161)

dylan-matthew-turner committed:
repos: update for kirkstone branch

We will want to perform upstream merges for kirkstone using the upstream
merge script. However, for this to work, repos.conf needs to be set up
to point to the kirkstone branches of upstreams which is the point of
this commit.

Signed-off-by: Dylan Turner <dylan.turner@ni.com>
